.github/containers/nox-cross-arch/arm64-ubuntu-20.04-python-3.11.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM docker.io/library/ubuntu:20.04
```

Comment:
The other tests in the CI use Ubuntu 20.04. It's better to be consistent here, and use the same base image instead of changing it to a different one.

Also, the test matrix should cover _both_ OS and python version. In case we get Ubuntu 22.04 controller, we need to be able to extend this systematically..

Reply:
Btw, there is no official image using ubuntu 20.04 with python 3.11. That's why this needs to be setup manually.